public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -103,6 +85,31 @@ define(function(require) {
     });
   };
 
+  // When there are file events triggered by the editor (or project is renamed), update the publish/republish status
+  Publisher.prototype.showUnpublishedChangesPrompt = function(callback) {
+    var publisher = this;
+
+    callback = callback || function() {};
+
+    if(publisher.needsUpdate) {
+      callback();
```

Comment:
you'll need to return here so that nothing in this function is run after the callback is called. Change this to:
```js
callback();
return;
```